src/otx/recipe/visual_prompting/openvino_model.yaml

```diff
@@ -5,7 +5,7 @@ model:
     model_name: segment_anything
     model_type: Visual_Prompting
     async_inference: False
-    use_throughput_mode: True
+    use_throughput_mode: False
```

Comment:
It's just a question. Why is this value set to `False`?

Reply:
Throughput mode only makes sense when async inference is executed, which is not the case for VPT now